src/content/docs/blog/2025/09/30/September 2025 project update/index.mdx

```diff
@@ -10,37 +10,37 @@ As we reach the end of September, it's time for another monthly update. For this
 
 ## Kernels
 
-AerynOS is currently running on 6.16.8. Whilst we are aware that Linux 6.17 was released on 28th September 2025, the AerynOS team takes the view that for larger software stacks, to hold back a "few" point releases after a major version update to ensure there are no unexpected issues is the wise thing to do for a team our size.
+AerynOS is currently running on 6.16.8. While we are aware that Linux 6.17 was released on 28th September 2025, the AerynOS team takes the view that for larger software stacks, the wise thing to do is to hold back for a few point releases after a major version update to ensure there are no unexpected issues.
 
-Whilst AerynOS is a rolling release distribution, we don't aim to be on the bleeding edge just for the sake of being on the bleeding edge.
+While AerynOS is a rolling release distribution, we don't aim to be on the bleeding edge just for the sake of being on the bleeding edge.
```

Comment:
```suggestion
Even though AerynOS is a rolling release distribution, we don't aim to be on the bleeding edge just for the sake of being on the bleeding edge.
```

Since this previous sentence started with "While".